src/Cake.ServiceFabric/Internal/ServiceFabricClusterConnection.cs

```diff
@@ -22,6 +23,13 @@ public ServiceFabricClusterConnection(ICakeEnvironment environment, IFileSystem
             _client = NotNull(client, nameof(client));
         }
 
+        public ServiceFabricClusterConnection(IFileSystem fileSystem, ICakeLog log, IFabricClient fabricClient)
+        {
+            _fileSystem = fileSystem;
+            _log = log;
+            this.fabricClient = fabricClient;
```

Comment:
@mholo65 I flip between the two, depending on the code base that I am in, but in this instance, I would agree 👍